modules/stackdriver-http-exporter/src/main/scala/io/janstenpickle/trace4cats/stackdriver/oauth/GoogleAccountParser.scala

```diff
@@ -11,20 +11,20 @@ import java.util.regex.Pattern
 
 import cats.syntax.either._
 import io.circe.Codec
-import io.circe.generic.extras.semiauto._
+import io.circe.generic.semiauto._
 
 object GoogleAccountParser {
   case class JsonGoogleServiceAccount(
     `type`: String,
-    projectId: String,
-    privateKeyId: String,
-    privateKey: String,
-    clientEmail: String,
-    authUri: String
+    project_id: String,
```

Comment:
Some of them seem unused, so they could be removed. Mirroring the snake_case of the input is the easiest way to work around the missing circe-generic-extras, but it's also possible to write a `Decoder` by hand (the format should basically never change) or try to pre-process the JSON before decoding. Let me know if you prefer a different approach.

Reply:
I ported these from another older library, so I don't think the extra fields are needed anymore.

I think this approach for snake case should be fine.